src/WebJobs.Script/Host/HostFunctionMetadataProvider.cs

```diff
@@ -116,14 +116,6 @@ private FunctionMetadata ReadFunctionMetadata(string functionDirectory, IFileSys
             }
         }
 
-        internal void ValidateName(string name, bool isProxy = false)
```

Comment:
Cleanup request from @mathewc during code review of binding names - this was not being used as there was a previous refactor to directly call `Utility.IsValidFunctionName(name)`.